### DISCUSSION TITLE
Misindexed points in gamma TiAl

### DISCUSSION THREAD

**Discussion by angus0997** [mtex-toolbox/mtex#2032] [Ask Anything]
Hi all,
I have a question about potential misindexed points in gamma TiAl.

I have attached an image of my IPF Z map. Some small grains are present as inclusions within other grains. I wonder if these are really different grains, or groups of misindexed points? In the image, I have coloured the boundaries of the grains of interest purple. 
![Picture1](https://github.com/mtex-toolbox/mtex/assets/101588689/46c5f40a-0519-4d93-8b32-131959b1d6fe)

These all have a misorientation angle of 90° to their host grain - shown in the misorientation angle distribution.
![Picture2](https://github.com/mtex-toolbox/mtex/assets/101588689/97f517e5-9a8a-4397-9924-97768ee53a8a)

I looked at the orientation relationship at these grain boundaries using:
```
mori_mean = mean(mori,'robust')
round2Miller(mori_mean)
```
And the result is: 
plane parallel:       (100) || (00-1)         
direction parallel:  [010] || [010]      
fit:                       0.628°

And this relationship is visualised using the crystal shape below (you can see the relationship in the purple and beige grains towards the bottom left of the map):
![Picture3](https://github.com/mtex-toolbox/mtex/assets/101588689/e3491e24-53d8-411e-9d3a-5bf6c4d78df0)

Is it possible that this apparent orientation relationship between the small grains and their hosts is the result of misindexing?
Cheers

**Comment by kilir** [comment]
Hi,
I don't know that material, but if it were systematic misindexing, I would't necessarily expect it to appear in such a patchy way but also much more single pixel misindexed points. Or by chance are those already removed but where there originally?
In some materials, twinning looks very much like that.
Cheers,
Rüdiger

**Reply by angus0997** [reply]
Hi Rüdiger,
I agree about the grains being "patchy", and also think this counts against mis-indexing. I removed all grains with fewer than 5 pixels, so some single pixel misindexing might have been removed; I will re-run the script with the grain size limit removed and see what effect it has.

I didn't know about the possibility of twins looking like this(!) - that is something for me to look into.
Cheers,
Angus

**Comment by ralfHielscher** [comment]
I also do not know this specific material, but I have seen similar maps and there it was due to misindexing. I would try to [merge](https://mtex-toolbox.github.io/GrainMerge.html) those grains.

Ralf.

**Reply by angus0997** [reply]
Hi Ralf,
Thanks for the reply. I agree with merging as a good option, _if_ one is sure that this is a result of misindexing. At the moment, I am not totally sure it is due to misindexing. Let me know if you have in mind somewhere I can see a similar map which is a result of mis-indexing.

Cheers,
Angus
